loan-events/.dockerignore

```
@@ -0,0 +1,3 @@
+node_modules
+dist
+forta.config.json
```

```suggestion
forta.config.json

```
nit: To be POSIX compliant, end the file with a newline